packages/snap/src/constants.ts

```diff
@@ -52,6 +52,6 @@ export const NETWORK_MAP = {
   'dev-testnet': DEV_TESTNET_URLS,
 }
 
-export const DEFAULT_NETWORK = 'mainnet';
+export const DEFAULT_NETWORK = 'testnet';
```

Comment:
We may forget to change this. Maybe you could change to mainnet in the PR and keep it as testnet locally only